src/users/serializers.py

```diff
@@ -76,6 +84,15 @@ def update(self, instance, validated_data):
                 instance.courses.add(c)
                 instance.save()
 
+        if 'minor' in validated_data:
+            instance.minor.clear()
+            minor_obj = validated_data.pop('minor')
+            for mnr in minor_obj:
+                dict(mnr)
```

Comment:
?

Reply:
It prevents the adding minors with the same name. It's pretty much identical to to code right above it for courses.

Reply:
oh I meant the dict noop

Reply:
Huh, I overlooked that. It might be a remnant of whatever stack overflow post I was referencing. Should I just remove the dict noops then merge?